gnomad_qc/v3/resources/release.py

```diff
@@ -60,7 +60,7 @@ def release_ht_path(
     if public:
         if het_nonref_patch:
             DataException("The patch HT will not be made public")
-        return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/{data_type}/gnomad.{data_type}.{version_prefix}{release_version}.sites.ht"
+        return f"gs://gcp-public-data--gnomad/release/{release_version}/ht/{data_type}/gnomad.{data_type}.{version_prefix}{release_version}.sites.ht"
```

Comment:
Just one small comment that no longer matters since this has been written. We should switch to write to `gnomad-public-requester-pays` and read from `gcp-public-data--gnomad` if it exists. This matters more for v4.

Reply:
Completely feel free to ignore, but now that I see this modification, I wonder if we should pull from https://github.com/broadinstitute/gnomad_methods/blob/b95578056f8051d00dcbb749a10e585d48888f82/gnomad/resources/grch38/gnomad.py#L263 if it exists for reading. I still think for writing we should go to gs://gnomad-public-requester-pays

Reply:
Thats a good idea, I wasnt thinking about the write integration, I'll update this!